frontend/packages/vkt/public/i18n/fi-FI/clerk.json

```diff
@@ -1,41 +1,102 @@
-          "registrationCloses": "Ilmottautuminen sulkeutuu"
+          "language": "Kieli ja taso",
+          "registrationCloses": "Ilmoittautuminen sulkeutuu"
         },
         "title": "Tutkintotilaisuudet",
         "toggleFilters": {
           "passed": "Menneet",
           "upcoming": "Tulevat"
         }
       },
+      "clerkExamEventOverview": {
+        "enrollmentDetails": {
+          "checkbox": {
+            "digitalCertificateConsent": "Annan suostumuksen siihen, että todistus toimitetaan minulle sähköisesti",
+            "oralSkill": "Suullinen taito",
+            "readingComprehensionPartialExam": "Tekstin ymmärtäminen",
+            "speakingPartialExam": "Puhuminen",
+            "speechComprehensionPartialExam": "Puheen ymmärtäminen",
+            "textualSkill": "Kirjallinen taito",
+            "understandingSkill": "Ymmärtämisen taito",
+            "writingPartialExam": "Kirjoittaminen"
+          },
+          "fields": {
+            "email": "Sähköposti",
+            "firstName": "Etunimet",
+            "lastName": "Sukunimi",
+            "phoneNumber": "Puhelinnumero",
+            "previousEnrollmentDate": "Päivämäärä"
+          },
+          "header": {
+            "contactDetails": "Yhteystiedot",
+            "digitalCertificateConsent": "Suostumus sähköisesti todistuksesta",
+            "nameAndSSN": "Nimi ja henkilötunnus",
+            "paymentReference": "Maksutunniste",
+            "previousEnrollment": "Aikaisempi osallistuminen valtionhallinnon kielitutkintoon",
+            "selectedSkills": "Valitut tutkinnot",
+            "selectedSubExams": "Valitut osakokeet"
+          },
+          "title": "Ilmoittautumisen tiedot"
+        },
+        "examEventDetails": {
+          "cancelUpdateDialog": {
+            "description": "Menetät tekemäsi muutokset.",
+            "title": "Haluatko poistua muokkausnäkymästä?"
+          },
+          "copyEmails": "Kopioi sähköpostit",
+          "downloadExcel": "Lataa Excel",
+          "enrollment": {
+            "fullExam": "Koko tutkinto"
+          }
+        },
+        "examEventDetailsFields": {
+          "header": {
+            "date": "Tutkintopäivä",
+            "isHidden": "Piilotettu",
+            "language": "Kieli",
+            "level": "Taso",
+            "registrationCloses": "Ilmoittautuminen sulkeutuu"
+          }
+        },
+        "examEventEnrollmentDetails": {
+          "cancelUpdateDialog": {
+            "description": "Menetät tekemäsi muutokset.",
+            "title": "Haluatko poistua muokkausnäkymästä?"
+          }
+        },
+        "examEventListingHeader": {
+          "canceled": "Peruutetut",
+          "expectingPayment": "Jonosta siirretyt / maksu puuttuu",
+          "paid": "Ilmoittautuneet",
+          "queued": "Jonoon ilmoittautuneet"
+        },
+        "toasts": {
+          "notFound": "Valittua tutkintotilaisuutta ei löytynyt",
+          "updated": "Tiedot tallennettiin"
+        }
```

Comment:
Nämä voisi ehkä kopioida `examEventEnrollmentDetails` alle ja vaihtaa `ClerkEnrollmentDetails.tsx` alla t:n keyprefiksi `vkt.component.clerkExamEventOverview.examEventEnrollmentDetails`. Nyt kun kahdessa paikkaa käytetään samaa lokalisaatiota, niin olisi ehkä parempi että ovat erit tai sitten olisivat common.json:in alla.